README.md

```diff
@@ -1,92 +1,74 @@
-<h1 align="center">Nimiq Validators</h1>
-
-<p align="center">
-  <a href="https://validators-api-mainnet.pages.dev">
-    <img src="./public/favicon.svg" alt="Nimiq Validators" width="64" />
+  <a id="README" href="#README" href="https://github.com/nimiq/core-rs-albatross/blob/albatross/README.md">
+    <img src="https://raw.githubusercontent.com/nimiq/developer-center/refs/heads/main/assets/images/logos/validators-API.svg" alt="Nimiq PoS Albatross Repository" width="600" />
   </a>
+</br>
+</br>
+
+[![Sync Mainnet](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml) [![Sync Testnet](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml)
 
-<p align="center">
-  Details of validators in the Nimiq Blockchain and their scores, calculated using Nimiq's VTS algorithm.
-<p>
+The Nimiq Validators API enables staking pools, based on single validators, to integrate with the Nimiq Wallet. This helps stakers make informed decisions when choosing where to stake their funds.
```

Comment:
Mmm, that is one of the main features it provides, but not the only one.

The validators API is also there to support other applications, such as nimiq.watch, so that those applications can get the data from a single source of truth.

This API is being used in multiple places already